### PR TITLE
feat(gitlab): display project stars only if there are

### PR DESCRIPTION
**Pull request by duboiss**
## Description
Improving the display of stars

## Screencast
**BEFORE**
CleanShot 2022-05-18 at 21 23 09@2x

**AFTER**
CleanShot 2022-05-18 at 21 22 31@2x

### CODE REVIEW

File: extensions/gitlab/src/components/project.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function ProjectListItem(props: { project: Project }): JSX.Element {
     <List.Item
       id={project.id.toString()}
       title={project.name_with_namespace}
-      subtitle={"Stars " + project.star_count}
+      subtitle={project.star_count ? `⭐ ${project.star_count}` : ""}
```
Comment (Loskir) [inline extensions/gitlab/src/components/project.tsx:33]:
```suggestion
      subtitle={project.star_count > 0 ? `⭐ ${project.star_count}` : ""}
```

**Review by Loskir — APPROVED** [review]
Great 👍

**Review by tonka3000 — APPROVED** [review]
👍

**Comment by duboiss** [comment]
Another idea would have been to also add the number of forks to identify the project even more easily, I don't know. On my side, currently, the number of stars is enough :)

**Review by pernielsentikaer — APPROVED** [review]
Let's get it merged 🔥 

Iif we add more data it could be an idea to move it to the [accessories property](https://developers.raycast.com/api-reference/user-interface/list#list.item.accessory) instead 😄

**Review by pernielsentikaer — COMMENTED** [review]
..And make some metadata images ;)

**Comment by Loskir** [comment]
That was fast…

**Pull request merged**